helm/charts/determined/templates/master-deployment.yaml

```diff
@@ -51,10 +51,10 @@ spec:
             {{- if .Values.masterMemRequest }}
             memory: {{ .Values.masterMemRequest  | quote }}
             {{- end}}
-      {{- if .Values.enterpriseEdition }}
+      {{- if .Values.imagePullSecretName}}
```

Comment:
Blocking: This looks ok to me, but I'm concerned about not being able to warn the user if they don't use the imagePullSecret in EE. Is this not a problem, or do we think it's sufficiently easy to diagnose a failed image pull without the warning? Should we add another line of logic to cover this?

Reply:
It's very easy to diagnose a failed image pull without a warning. Even if you add the previous check, we can't validate the secret so the check didn't really help a lot.